server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -251,6 +251,11 @@ it is around the label */
     @include u-padding-top(4);
     @include u-padding-bottom(2);
   }
+
+  // Hide the government name on desktop (if set to hidden). This class is only applied when the HIDE_CIVIC_ENTITY_NAME_IN_HEADER flag is enabled
+  .cf-gov-name {
```

Comment:
Instead of creating custom classes can you use the `.display-none` / `.tablet:display-block` USWDS [utility classes](https://designsystem.digital.gov/utilities/display)?